core/src/main/scala/uk/gov/nationalarchives/tdr/api/core/db/DbConnection.scala

```diff
@@ -1,21 +1,18 @@
 package uk.gov.nationalarchives.tdr.api.core.db
 
+import com.typesafe.config.{ConfigFactory, ConfigValueFactory}
 import slick.jdbc.PostgresProfile.api._
 
 object DbConnection {
-  private val driver = "org.postgresql.Driver"
-  // Load PostgreSQL driver into classpath
-  Class.forName(driver)
-
   private val dbConfig = sys.env.get("TDR_API_ENVIRONMENT") match {
     case Some("TEST") => PrototypeDbConfig
     case _ => DevDbConfig
   }
 
-  val db = Database.forURL(
-    url = dbConfig.url,
-    user = dbConfig.username,
-    password = dbConfig.password,
-    driver = driver
-  )
+  private val config = ConfigFactory.empty()
```

Comment:
This is my bad naming, because I did this in a rush, but could we distinguish between the two `config` variables? Maybe the one on line 7 should be `connectionParameters` and this one can be `postgresConfig`?

Reply:
Resolved